core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -143,6 +144,9 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
       .orElse(sparkProperties.get("spark.master"))
       .orElse(env.get("MASTER"))
       .orNull
+    commonExtraClassPath = Option(commonExtraClassPath)
```

Comment:
The change in this file is not technically necessary, unless you add a specific command line option for this config (which I don't think you should). See how it doesn't have a field for the executor's extra class path (because there's no command line option for it).